.github/workflows/ci.yaml

```diff
@@ -90,3 +90,46 @@ jobs:
         run: tox run -e ${{ matrix.tox-environments }} -- -m '${{ steps.select-tests.outputs.mark_expression }}'
         env:
           CI_PACKED_CHARMS: ${{ needs.build.outputs.charms }}
+
+  integration-test-ha:
+    strategy:
+      fail-fast: false
+      matrix:
+        tox-environments:
+          - integration-ha-ha
```

Comment:
**question:** What is this? Is it so it can be nested?

Reply:
They will look different, just a placeholder at the moment, see [following PR](https://github.com/canonical/kafka-operator/pull/129/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR99)